src/Service.Host/views/PlCreditDebitNote.cshtml

```diff
@@ -190,10 +190,10 @@
             </div>
 
             @{
-                if ((@Model.NoteType.Type == "D") && (@Model.PurchaseOrder?.DocumentType.Name != "RO"))
+                if ((@Model.NoteType.Type == "D") && (@Model.PurchaseOrder?.DocumentTypeName != "RO"))
```

Comment:
can probs lose the ? after the PurchaseOrder since we always have a PO here

Reply:
also a bit of redundant parenthesis on this line?

if (@Model.NoteType.Type == "D")&& @Model.PurchaseOrder?.DocumentTypeName != "RO")

is cleaner